zenodo/modules/records/serializers/dcat.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
```

Comment:
```suggestion
# This file is part of Zenodo.
```

Similar mentions below should be updated as well. If this was copy-pasted, check other file(s) as well.